aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/AddVersionBuilder.java

```diff
@@ -596,5 +602,9 @@ public String getAipConsoleUsername() {
         public int getTimeout() {
             return configuration.getTimeout();
         }
+
+        public boolean isVerbose() {
```

Comment:
remove all these isVerbose() method in each class, doesnt make sense

Reply:
How I access the global config ârameter then?

Reply:
if u can access to configuration, you can have configuration.isVerbose

Reply:
getDescriptor().isVerbase() gives the same results.
But however if is that you want I'll do